server/base/src/main/java/org/apache/accumulo/server/util/MasterMetadataUtil.java

```diff
@@ -239,12 +238,7 @@ public static StoredTabletFile updateTabletDataFile(ServerContext context, KeyEx
     }
     tablet.putFlushId(flushId);
 
-    if (mergeFile != null) {
-      tablet.deleteFile(mergeFile);
-    }
-
     unusedWalLogs.forEach(tablet::deleteWal);
-    filesInUseByScans.forEach(tablet::putScan);
```

Comment:
This related to removal for merging minor compactions?

Reply:
yeah, eliminating merging minor compaction eliminated the need of dealing with a file being merged and in use by scans.